prometheus_kafka_consumer_group_exporter/__init__.py

```diff
@@ -107,7 +107,10 @@ def main():
             consumer_config.update(converted_config)
 
     if args.bootstrap_brokers:
-        consumer_config['bootstrap_servers'] = args.bootstrap_brokers.split(',')
+        logging.info("Overriding 'bootstrap_server' with command line '-b' argument")
```

Comment:
Lets drop this logging for now. It's reasonable to expect that if you provide a value in a CLI argument it'll override a config file/default value.

Reply:
Perfect! Fixed in 6d939db, I've re-pushed the changes without that line to have a single commit with the changes...